ow_plexil/src/plans/ReferenceMission1.plp

```diff
@@ -76,11 +77,11 @@ ReferenceMission1: Concurrence
     log_info ("Starting Reference Mission 1, Sol 0 ...");
     LibraryCall ImageLandingSite (InstanceName = "ReferenceMission1",
                                   IgnoreCrash = true);
-    LibraryCall Unstow();
```

Comment:
Why was this removed?  Unstow is required before doing anything else with the arm.

Reply:
I removed it because it was blocking the camera's view, but I forgot to add an unstow into the IdentifySampleTarget before the guarded move. I will make that fix!